docs/getting-started/quickstart.md

```diff
@@ -19,7 +21,245 @@ Let's create your first DBOS app! You develop this app in a folder on your compu
   }}
 />
 
-In this guide, we start by deploying a sample "Hello" app to the cloud. After that, we show you how to run it locally.
+In this guide, you'll start by deploying a sample "Hello" app to the cloud. Then, you'll run it locally.
+
+
+<LargeTabs groupId="language">
```

Comment:
I think we should put TS first, Python should be prominent but not dominant.
(For science, it would be even more fun to have it 50/50 so we get a fair sense of how many people go which way.)
Right now TS has debugging, performance, libraries, more demos, and a bunch of other stuff python doesn't have, so if we want our best foot forward it is still TS for the next while.

Reply:
We'll talk about this offline